Traceback (most recent call last):
  File "/lfs/l7/hess/users/spencers/realdata/testscript.py", line 12, in <module>
    runnumber = int(sys.argv[1])
IndexError: list index out of range
